README.md

```diff
@@ -20,19 +20,40 @@ module.exports = {
     {
       resolve: `gatsby-plugin-material-ui`,
       // If you want to use styled components, in conjunction to Material-UI, you should:
-      // - Change the injection order
+      // - Change the injection order by adding a gatsby-mui-config.js and specifying the path in pluginConfigModule (e.g. src/utils/gatsby-mui-config.js)
       // - Add the plugin
+      // Use either injectFirst or customInPoint (not both)
       options: {
-        // stylesProvider: {
-        //   injectFirst: true,
-        // },
+        injectStyles: true,
+        pathToConfigModule: 'src/utils/gatsby-mui-config',
       },
       // 'gatsby-plugin-styled-components',
     },
   ],
 };
 ```
 
+Add `gatsby-mui-config.js`:
+
+```javascript
+const gatsbyMuiConfig = {
+  // createGenerateId: () => () => {},
+  // plugins: [],
+  injectFirst: true,
+  insertionPoint: 'jss-insertion-point',
```

Comment:
It's probably something we would want to move in @material-ui/styles directly rather than having custom options for Gatsby.